scripts/screeps_rl_experiment_card.py

```
@@ -203,18 +209,214 @@ def safety_block() -> JsonObject:
+def multi_tier_scenario_fixture_summary(path: Path) -> JsonObject:
+    """Validate and summarize the active hostile fixture for the multi-tier scenario."""
+    fixture_path = path.expanduser()
+    try:
+        raw_bytes = fixture_path.read_bytes()
+        raw = json.loads(raw_bytes.decode("utf-8"))
+    except OSError as error:
+        raise CardValidationError(f"multi-tier scenario fixture is not readable: {fixture_path}: {error}") from error
+    except (UnicodeDecodeError, json.JSONDecodeError) as error:
+        raise CardValidationError(f"multi-tier scenario fixture is not valid JSON: {fixture_path}: {error}") from error
+    if not isinstance(raw, dict):
+        raise CardValidationError("multi-tier scenario fixture must be a JSON object")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)